### ISSUE TITLE
No structures generating

### ISSUE THREAD

**Issue by Rulenine** [stal111/Valhelsia-Structures#39]
Hi all,

I am running the 16.0.2 mod version on Forge 32.0.108. The mod is definitely working, because I can see items such as the brazier, but no structures generate. No castles or castle ruins anywhere. I've traveling via creative multiple thousands of blocks in each direction and found nothing except standard villages. Is there anything I'm missing? Are there any known compatibility issues with this mod? Would really appreciate the help, as this mod looks perfect for my server!

**Comment by ScottKillen** [comment]
Probably related to https://github.com/Glitchfiend/BiomesOPlenty/issues/1691

Perhaps update BOP?

**Comment by Rulenine** [comment]
> 
> 
> Probably related to [Glitchfiend/BiomesOPlenty#1691](https://github.com/Glitchfiend/BiomesOPlenty/issues/1691)
> 
> Perhaps update BOP?

No BOP installed. I have Quark, Atmosphere, Autumnity, Towers of the Wild, and a few small mods that add some items but do not adjust anything related to worlgen, biomes, or that sort of thing.

**Comment by Psykrot** [comment]
Also having the same issue. However, I'm on 16.0.3 and using the latest version of Biomes O' Plenty (12.0.0.409 atm). The mod page states that it works with Biomes O' Plenty specifically so I don't think that is the issue. 

I tried turning the spawn_chance up to 1 for everything and the flatness_delta to 0, 4, and 64, but nothing spawns every time I load a new world. 

It might be a mod compatibly issue. Are you using any of the following mods?: YUNG's Better Caves (Forge), MineColonies/Structurize, or Immersive Engineering. Those are the only mods in my pack that I think might conflict with the structure spawning.

**Comment by Vaelzan** [comment]
@Psykrot Please make a new issue for yours since it is a completely different Minecraft version.

However, to all people having problems: Have you tried using the /locate command to search for structures, or are you just trying to find them by flying around? They can be rare and it's easy to miss them.

**Comment by Psykrot** [comment]
> @Psykrot Please make a new issue for yours since it is a completely different Minecraft version.
> 
> However, to all people having problems: Have you tried using the /locate command to search for structures, or are you just trying to find them by flying around? They can be rare and it's easy to miss them.

@Vaelzan Sorry for the wrong version, /locate command worked for the versions I'm on. I'm gonna have to mess with the settings to get structures to spawn closer together. Thanks!

**Comment by mcquestion** [comment]
How would one up the spawn rates? Merry Christmas btw guys! :)

**Comment by mrmyers365** [comment]
Hey! Try going to the config folders in .minecraft, and locate both client and server Valhesia Structures folders. If you check both, you will find that the client one is empty, while the server has all the codes you'll need for the mod to work properly. What I did was to delete the empty "client" folder, and then changed the name of the server one into "client". Then, I used the command /locate valhesia and teleport to the locations given and everything was perfect. This worked for me and I hope that it will work for you too!
